draft-lenders-dns-over-coap.md

```diff
@@ -147,8 +147,8 @@ A DoC client encodes a single DNS query in one or more CoAP request
 messages using either the CoAP GET {{!RFC7252}}, POST {{!RFC7252}}, or
 FETCH {{!RFC8132}} methods. Requests of either method type SHOULD include
 an Accept option to indicate the type of content that can be parsed in the
-response. A client MUST be able to parse messages of Content-Format
-"application/dns-message" regardless of the provided Accept option.
+response. A client MUST eventually request with Accept
+"application/dns-message" and be able to parse such a response.
```

Comment:
I don't think the Accept option is strictly necessary ("MUST […] request with Accept [option?]", however, can be read as such) and as it stands it somewhat contradicts the sentence before "[…] SHOULD include an Accept option […]".